### PR TITLE
Add a Design Tokens page to the FreeStyle docs

### PR DESCRIPTION
**Pull request by paddyduke**
This section documents some of our general visual/style rules for designing UI patterns in FreeAgent, and provides some reference scales for some commonly used properties.

**NB:** I recommend checking out the branch and having a look when you're reviewing. It's hard to get a feel for the additions from the code alone.

### CODE REVIEW

File: docs/guidance/design-tokens.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+---
+categories: guidance
+layout: page
+title: Design Foundations
```
Comment (paddyduke) [inline docs/guidance/design-tokens.md]: I'm not dying about this name, but 'Design Tokens' seems a bit specific. I feel like this maybe isn't clear enough that it's entirely about visuals, and not our broader design principles.

Happy to take suggestions here.

File: docs/guidance/design-tokens.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+---
+
+These are the basic visual foundations of our design system, used in all of our UI design. They consist of general rules and scales for values such as spacing, rounded corners, shadows, and more.
+
+The rules here often use terminology from CSS, but they are not written explicitly in CSS or any other specific language. They should be used as a reference both when designing in Sketch or when writing styles in any format.
+
+* Will be replaced with the ToC, excluding the "Contents" header
+{:toc}
+
+## General Rules
```
Comment (paddyduke) [inline docs/guidance/design-tokens.md]: These possibly belong elsewhere in the long term.

**Comment by summers1981FA** [comment]
Hey @paddyduke, looks good. I really like the use of the graphics in the section and how simple it will be for a designer to visually see how it should look and grab the values easily. 

As discussed, there are some points I wasn't sure we might need to agree on as team:

1) Should we name it 'Design Foundations' when other systems name it 'Design Tokens'. Pretty new concept but just wondered whether we should keep it consistent with name we know it as and possibly in the industry?
2) Is it worth pulling this section out and making it a top level item then create sub pages (spacing, border-radius etc). This section could possibly expand with the likes of Time, Speed, Line-height, Font etc. I am basing this theory on Lightening Design System.
3) The part of the page for me that doesn't look as visually pleasing is where the values are listed (ex. 2,4,8,16,24 etc). I wonder if we could lose the height of the page by putting these values into a table in some way?
4) I wasn't sure if the general rules and tokens should be on the same page. Not sure if we could possibly re-organise?

**Comment by summers1981FA** [comment]
LGTM @paddyduke. 

Possibly as another branch I think we have some scope to possibly tidy up this list (inline?) to remove the height of the page
screen shot 2018-02-07 at 13 48 39

File: assets/scss/docs/components/_table.scss [hunk 3]
```diff
@@ -0,0 +1,336 @@
...
+
+/**
+* Style a parent table header row so that it appears more subdued than a regular
+* table header.
+*/
+
+.Table .Table-row--parentHeader th {
+  background-color: #f1f7fa;
+  font-weight: normal;
+  color: #006199;
```
Comment (summers1981FA) [inline assets/scss/docs/components/_table.scss]: alphabetical order

File: assets/scss/docs/components/_table.scss [hunk 4]
```diff
@@ -0,0 +1,336 @@
...
+
+/**
+* Style a table row in a particular way when it's a parent; when it has
+* collapsable child row belows it, as in the P&L tables.
+*/
+
+.Table .Table-row--parent {
+  background-color: #f1f7fa;
+  font-weight: 600;
+  color: #174f73;
```
Comment (summers1981FA) [inline assets/scss/docs/components/_table.scss]: alphabetical order

File: assets/scss/docs/components/_table.scss [hunk 5]
```diff
@@ -0,0 +1,336 @@
...
+*/
+
+.Table--desktop {
+  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+}
+
+.Table--borderless td {
+  padding-bottom: 4px;
+  padding-top: 4px;
+  border-top: 0;
```
Comment (summers1981FA) [inline assets/scss/docs/components/_table.scss]: alphabetical order

**Comment by paddyduke** [comment]
I've fixed the alphabetisation issues and also removed some of the less useful examples from the tables.

**Pull request merged**